bin/create_bag.rb

```diff
@@ -12,11 +12,14 @@
 require 'rubygems'
 require 'bundler/setup'
 require 'moab/stanford'
+require 'yaml'
 
 include Stanford # rubocop:disable Style/MixinUsage
 
+settings = YAML.load_file('../config/environments/prod.yml')
```

Comment:
any chance it ever gets run from a non-prod env?

Reply:
No, this is always used to fetch bags from the "prod" storage roots.